src/NovelRT/Audio/AudioService.cpp

```diff
@@ -2,6 +2,7 @@
 // for more information.
 
 #include <NovelRT.h>
+#include <NovelRT/Audio/AudioService.h>
```

Comment:
I'd probably remove the NovelRT.h include here unless you absolutely need it. Seems redundant to have both.

Reply:
It's legacy code, but I can adjust it since we're not dependent on novelrunner anymore

Reply:
Done